lib/reducers/clock/index.js

```diff
@@ -0,0 +1,10 @@
+export function clock(state = {}, action) {
+    switch (action.type) {
+        case 'GET_TIME':
```

Comment:
It might be best to reference the constants used for the action creators (RE development best practice)